src/app/hero.ts

```diff
@@ -0,0 +1,4 @@
+export class Hero {
```

Comment:
If we define a simple `type` not a `class` object, please avoid using `class` definition but just use `interface` or `type` definition.

Reply:
export interface Hero {